### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by Thi-Ponce**
## Learning objectives
- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,16 @@
...
+      </div>
+      <div class="user-info">
+        <p class="user-name">
+          <%= link_to user.name, user_path(id: user.id) %>
+        </p>
+        <p class="user-numbers">Number of Posts: <%= user.posts_counter || 0 %></p>
+      </div>
+  </div>
+  <% end %>
+</section>
```
Comment (ahmetbozaci) [inline app/views/users/index.html.erb:16]: - [ ] It is not a problem if ı click on the first post 1 but if ı click on the post 3 of the page ı am getting an error. Please fix it.
image
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @Thi-Ponce  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: app/views/users/index.html.erb (hunk 1 above)
Comment (Thi-Ponce) [reply app/views/users/index.html.erb:16]: Hello @ahmetbozaci , thank you for the review.

I just created a new user with 3 posts and when I go in all of them I don't get that error, could you tell me how you got yours? 
pic3
index
pic1
pic2

Comment (ahmetbozaci) [reply app/views/users/index.html.erb:16]: @Thi-Ponce
**I create these samples and ı get error when click post 4 and post 3.  I dont know why. It has not problem when click post 1 and post 2**
```
first_user = User.create(name: 'Tom', 
  photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80', 
  bio: 'Teacher from Mexico.', posts_counter:0)
second_user = User.create(name: 'Lilly', 
  photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80', 
  bio: 'Teacher from Poland.', posts_counter:0)
  first_post = Post.create(author: first_user, title: 'Post 1', text: 'This is my first post',comments_counter:0, likes_counter:0)
  second_post = Post.create(author: second_user, title: 'Post 2', text: 'This is my second post',comments_counter:0, likes_counter:0)
  third_post = Post.create(author: first_user, title: 'Post 3', text: 'This is my third post',comments_counter:0, likes_counter:0)
  fourth_post = Post.create(author: second_user, title: 'Post 4', text: 'This is my fourth post',comments_counter:0, likes_counter:0)
  Comment.create(post: first_post, author: second_user, text: 'Hi Tom!' )
  Comment.create(post: second_post, author: first_user, text: 'Hi Tom!' )
  Comment.create(post: second_post, author: second_user, text: 'Hi Tom!' )
  Comment.create(post: third_post, author: first_user, text: 'Hi Tom!' )
  Comment.create(post: third_post, author: second_user, text: 'Hi Tom!' )
  Comment.create(post: third_post, author: second_user, text: 'Hi Tom!' )
  ```

Comment (Thi-Ponce) [reply app/views/users/index.html.erb:16]: Hey @ahmetbozaci , I just ran the commands you sent the screenshot and it work normally on my end, I really don't know what to do.

Comment (ahmetbozaci) [reply app/views/users/index.html.erb:16]: > Hey @ahmetbozaci , I just ran the commands you sent the screenshot and it work normally on my end, I really don't know what to do.

@Thi-Ponce If everything works fine please send a review again. The next code reviewer can check.

**Review by AshakaE — APPROVED** [review]
Hi @Thi-Ponce ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @Thi-Ponce  👋,

_This is an additional review as a result of the second opinion request._

This was supposed to be a second opinion review, but I am glad to see that it was approved now and everything went fine.

Good luck with your future projects!

Keep rocking 🚀!

**Pull request merged**